sys/include/net/gnrc/nettype.h

```diff
@@ -77,7 +77,8 @@ typedef enum {
 #ifdef MODULE_GNRC_TCP
     GNRC_NETTYPE_TCP,           /**< Protocol is TCP */
 #endif
-#ifdef MODULE_GNRC_UDP
+#if defined(MODULE_GNRC_UDP) || \
+    (defined(MODULE_GNRC_SIXLOWPAN_IPHC_NHC) && defined(MODULE_GNRC_SIXLOWPAN_ND_BORDER_ROUTER))
```

Comment:
I have to test here for udp || iphc_nhc, for scenarios where we want to use udp but **not** iphc_nhc (this is `nettype.h`)